Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM ubuntu:xenial-20210804
+FROM ubuntu:focal-20211006
 ARG PYTHON_VERSION=2.7
 
 RUN groupadd -g 5000 sync-engine \
   && useradd -d /home/sync-engine -m -u 5000 -g 5000 sync-engine
 
-RUN DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get dist-upgrade -y && apt-get install -y \
+ENV TZ="Etc/GMT"
```

Comment:
Had to set this and install tzdata as a separate `apt-get install` to prevent it from asking the timezone.